test/fixtures/expected.js

```diff
@@ -0,0 +1,12 @@
+'use strict'
+
+const CID = require('cids')
+
+module.exports = {
+  'array-link': new CID('zdpuAsZ8roJvCazJFXUnSULspbXfxrLkUkNxy6SToaiDpEiap'),
```

Comment:
maybe all these properties that need quoting should just be converted to camelcase because the quoting gets a bit awkward when they're used: `expectedCIDs['array-link']` -> `expectedCIDs.arrayLink` which also parallels `arrayLinkCBOR` which it's compared to, as well as `arrayLinkJS`.